docs/pages/guides/monorepos.mdx

```diff
@@ -62,16 +62,20 @@ Before we create our app, we have to create the **apps** directory. This directo
 
 After copying or creating the first app, run `yarn` to check for common warnings.
 
-#### Modify the Metro config
+#### Configure Metro
 
-Expo's Metro config has monorepo support for bun, npm, yarn, and pnpm. You don't have to manually configure Metro when using monorepos if you use the config from [`expo/metro-config`](/guides/customizing-metro/). If that's the case, you can skip this step.
+Expo's Metro config has monorepo support for bun, npm, yarn, and pnpm. By default, if you use Expo CLI and do not have a **metro.config.js** file, you be will using the config from `expo/metro-config`, and you can proceed to the next step. 
 
-To configure a monorepo with Metro manually, there are two main changes:
+[If you customize your **metro.config.js**](/guides/customizing-metro), we strongly recommend that you extend `expo/metro-config` to get the best experience with monorepos and Expo tools in general.
+
+If for some reason this is not possible, or if you extend `expo/metro-config` and want to ensure that you are not overwriting configuration that is relevant to monorepos, there are two main configuration changes to be aware of:
 
 1. Make sure Metro is watching all relevant code within the monorepo, not just **apps/cool-app**.
 2. Tell Metro where it can resolve packages. They might be installed in **apps/cool-app/node_modules** or **node_modules**.
 
-We can configure this by [creating a **metro.config.js**](/guides/customizing-metro/#customizing) with the following content:
+<Collapsible summary="Example metro.config.js">
+
+The following **metro.config.js** demonstrates configuration changes `expo/metro-config` applies automatically relevant to monorepos, to demonstrate what is happening under the hood. You do not need to copy this code into your project, but use it as a reference when you are customizing your **metro.config.js** if you change any of the config properties referenced below.
```

Comment:
```suggestion
The following **metro.config.js** demonstrates configuration changes `expo/metro-config` applies automatically relevant to monorepos. You do not need to copy this code into your project, but use it as a reference when you are customizing your **metro.config.js** to change any config properties referenced below.
```

I don't think we need the part "...happening under the hood." The next sentence clearly specifies that the code is for reference.